src/ngp_algorithms/WallFuncGeometryAlg.C

```diff
@@ -108,7 +108,8 @@ void WallFuncGeometryAlg<BcAlgTraits>::execute()
         DoubleType ypBip = 0.0;
         for (int d=0; d < BcAlgTraits::nDim_; ++d) {
           const DoubleType nj = v_area(ip, d) / aMag;
-          const DoubleType ej = 0.25 * (v_coord(nodeR, d) - v_coord(nodeL, d));
+        //const DoubleType ej = 0.25 * (v_coord(nodeR, d) - v_coord(nodeL, d));
+          const DoubleType ej = 1.0 * (v_coord(nodeR, d) - v_coord(nodeL, d));
```

Comment:
May be this `1.0` (and previously 0.25) should be some variable so that this can be used consistently in unit tests.

For example, in the header 

```
static constexpr DoubleType wallNormalHeightFactor = 1.0;
```